### PR TITLE
[e2e-ingress] Get node tag from instance under GKE

### LINKED ISSUE #48167: Failure cluster [1fb2ba...] failed 73 builds, 3 jobs, and 1 tests over 1 days
##### Error text:
```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/ingress.go:142
Expected error:
    <*errors.errorString | 0xc4200d5200>: {
        s: "incorrect target tags [gke-bootstrap-e2e-b0c71c8c-node], expected [k8s-bootstrap-e2e-node]: found differences: map[gke-bootstrap-e2e-b0c71c8c-node:{}]",
    }
    incorrect target tags [gke-bootstrap-e2e-b0c71c8c-node], expected [k8s-bootstrap-e2e-node]: found differences: map[gke-bootstrap-e2e-b0c71c8c-node:{}]
not to have occurred
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/ingress.go:131
```
##### Failure cluster statistics:
1 tests failed,    3 jobs failed,    73 builds failed.
Failure stats cover 1 day time range '26 Jun 2017 23:52 UTC' to '27 Jun 2017 23:52 UTC'.
##### Top failed tests by jobs failed:

| Test Name | Jobs Failed |
| --- | --- |
| [k8s.io] Loadbalancing: L7 [k8s.io] GCE [Slow] [Feature:Ingress] should create ingress with given static-ip | 3 |

##### Top failed jobs by builds failed:

| Job Name | Builds Failed | Latest Failure |
| --- | --- | --- |
| ci-kubernetes-e2e-gci-gke-ingress | 37 | [27 Jun 2017 22:53 UTC](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gke-ingress/10507) |
| ci-kubernetes-e2e-gke-ingress | 33 | [27 Jun 2017 23:06 UTC](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-ingress/10453) |
| ci-kubernetes-e2e-ubuntu-gke-ingress | 3 | [27 Jun 2017 12:53 UTC](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-ubuntu-gke-ingress/1008) |

<details><summary>Rationale for assignments:</summary>

| Assignee or SIG area | Owns test(s) |
| --- | --- |
| eparis | [k8s.io] Loadbalancing: L7 [k8s.io] GCE [Slow] [Feature:Ingress] should create ingress with given static-ip |

</details><br>

[Current Status](https://go.k8s.io/triage#1fb2bacd1b1f35178bd6)

### PR DESCRIPTION
**Pull request by MrHohn**
**What this PR does / why we need it**: Making ingress CI green again.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #48167 

**Special notes for your reviewer**:
/assign @nicksardo 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/framework/firewall_util.go [hunk 1]
```diff
@@ -124,6 +124,13 @@ func SetInstanceTags(cloudConfig CloudConfig, instanceName, zone string, tags []
 	return resTags.Items
 }
 
+// GetNodeTags gets k8s node tag from one of the nodes
+func GetNodeTags(c clientset.Interface, cloudConfig CloudConfig) []string {
+	nodes := GetReadySchedulableNodesOrDie(c)
+	Expect(len(nodes.Items) > 0).Should(BeTrue())
+	return GetInstanceTags(cloudConfig, nodes.Items[0].Name).Items
```
Comment (MrHohn) [inline test/e2e/framework/firewall_util.go]: @nicksardo Changed to return all tags instead.

**Comment by MrHohn** [comment]
Thanks!

/assign @bowei 
For approval.

File: test/e2e/framework/firewall_util.go [hunk 2]
```diff
@@ -124,6 +124,13 @@ func SetInstanceTags(cloudConfig CloudConfig, instanceName, zone string, tags []
 	return resTags.Items
 }
 
+// GetNodeTags gets k8s node tag from one of the nodes
+func GetNodeTags(c clientset.Interface, cloudConfig CloudConfig) []string {
+	nodes := GetReadySchedulableNodesOrDie(c)
+	Expect(len(nodes.Items) > 0).Should(BeTrue())
```
Comment (bowei) [inline test/e2e/framework/firewall_util.go]: If len(nodes.Items) == 0, can't we just return []string{}?

Comment (MrHohn) [inline test/e2e/framework/firewall_util.go]: Sounds good, added logging for 0 node and return []string{}.

**Pull request merged**